search-sentiments-map/README.md

```diff
@@ -1,21 +1,40 @@
-This web application visualizes trending google search results geographically and performs sentiment analysis on the search results. It utilizes the Maps API, Charts API, Google Trends API, and Google Custom Search APIs. It also utilizes Java Servlets.
+This web application visualizes trending google search topics and their results geographically and performs sentiment analysis on the search results. 
+It is build with NodeJS and it utilizes the Google Maps Javascript API, Google Charts API, Google Custom Search API, and the NodeJS Google Trends API.
+
+Commands:
 
-Commands to run:
 1. Install node_modules with
 `npm install`
-2. Run the server on localhost:3000 with
+
+2. Run the server on localhost:4503 with
 `npm start`
-3. Deploy to trending-search-sentiments.appspot.com with
+
+3. Deploy to "trending-search-sentiments.appspot.com" with
 `gcloud app deploy`
 
-To set project ID: 
+Need to have project ID set before deploying or running: 
 `gcloud config set project PROJECT_ID` or `export GCLOUD_PROJECT=...`
 
+Need to have an .env file to run with custom search. File format:
+SEARCH_API_KEY=key
+
+To update the Datastore indices:
+`gcloud app deploy index.yaml`
+
 To update the app engine's cron job:
 `gcloud app deploy cron.yaml`
 
-Need to add .env file to run. File format:
-SEARCH_API_KEY=key
+The cron job that updates trending search topics and their search results is currently set up to run every day at 11am and 11pm, New York time.
```

Comment:
Change it to be Eastern Time, not New York time.

Reply:
New York is what cron.yaml uses as the timezone, so I'm putting that here. Also New York is more specific than Eastern cuz you can have eastern in any country

Reply:
Actually Eastern Time(ET) is a time zone. There is only one is Eastern Time. New York is part of Eastern Time. I believe the cron.yaml also says "Eastern Time"(if you check https://pantheon.corp.google.com/appengine/cronjobs?project=trending-search-sentiments that's how it categorizes the time). New York is not mentioned at all. New York time is also not a "timezone", it is the time in a city. That would be like saying "LA Time" instead of "Pacific Time"(which is the timezone).

Reply:
I was talking about in the cron.yaml file we have " timezone: America/New_York"

Reply:
But yeah I can put Eastern Time if you feel strong about it